installer/cli/uninstallcli.py

```diff
@@ -14,12 +14,21 @@
 )
 @click.option('--aws_access_key', prompt=True, envvar='AWS_ACCESS_KEY_ID')
 @click.option('--aws_secret_key', prompt=True, envvar='AWS_SECRET_ACCESS_KEY')
-def uninstall(mode, aws_access_key, aws_secret_key):
+@click.option(
+    "--region",
+    "-r",
+    help='AWS region to uninstall. Only the listed choices have been verified',
+    type=click.Choice(['us-east-1', 'us-west-2']),
+    envvar='AWS_DEFAULT_REGION',
+    prompt=True
+)
+def uninstall(mode, aws_access_key, aws_secret_key, region):
     """Uninstalls the Jazz Stack."""
     click.secho('\n\nConfiguring uninstall', fg='blue')
     click.secho('\nLogging to `uninstall.log`', fg='green')
     os.environ['AWS_ACCESS_KEY_ID'] = aws_access_key
     os.environ['AWS_SECRET_ACCESS_KEY'] = aws_secret_key
+    os.environ['AWS_DEFAULT_REGION'] = region
```

Comment:
Reason for setting/overwriting people's env vars on uninstall?

Reply:
For uninstall, it will throw error when there is no configured aws.

> botocore.exceptions.NoRegionError: You must specify a region.

For install, it is handled in update_main_terraform_vars - common.py https://github.com/tmobile/jazz-installer/blob/b9a0dc1f6b70cc7f67320f92178d830853eee024/installer/configurators/common.py#L61

Reply:
We invoke `boto3`, we should be able to pass it the region variable directly.

The installer should read people's env vars, it should not set them, especially if we can simply pass the value we already have to the thing that needs it, without using env vars as a proxy when it's not really needed.

Reply:
@bleggett yeah, replaced with the installed region and assigned as `os.environ['AWS_DEFAULT_REGION'] = get_terraform_output_var("region")`

Hence, during uninstall there is no user input for region and we are reusing the installed region as it is. We don't need to pass region in all cases ie, boto3 and aws cli commands, if we set the AWS_DEFAULT_REGION. Also we can reduce the code to a single line.

Reply:
Yep, I get that it would be simpler if we can forcibly override the env var, but it's not a good user experience. If I'm working on other things in other AWS accounts and this installer stomps on my env vars I'm going to be annoyed with it.